test/scripts/e2e-kind.sh

```diff
@@ -47,6 +47,16 @@ should resolve connection reset issue #74839
 # TO BE FIXED BY https://github.com/kubernetes/kubernetes/pull/129049
 Services should be able to switch session affinity for NodePort service
 
+# TO BE FIXED BY https://github.com/kubernetes/kubernetes/issues/130434
+should update the ephemeral containers in an existing pod 
```

Comment:
why is https://github.com/kubernetes/kubernetes/issues/130434 causing this issue around ephemeral containers? from glancing at 130434 it sounds like the test fails on azure which currently looks closed.. not sure what the resolution was?